src/main/java/com/google/devtools/build/lib/analysis/test/TestConfiguration.java

```diff
@@ -281,6 +281,14 @@
         help = "If true, then Bazel will run coverage postprocessing for test in a new spawn.")
     public boolean splitCoveragePostProcessing;
 
+    @Option(
+        name = "use_target_platform_for_tests",
+        defaultValue = "false",
+        documentationCategory = OptionDocumentationCategory.EXECUTION_STRATEGY,
+        effectTags = {OptionEffectTag.EXECUTION},
+        help = "If true, then Bazel will run coverage postprocessing for test in a new spawn.")
```

Comment:
Is it possible that this `help` is a copy-paste artifact? It looks to me in need of updating.